autoload/lightline/colorscheme/papercolor_dark.vim

```diff
@@ -0,0 +1,257 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/autoload/lightline/colorscheme/papercolor_light.vim
+" Author: TKNGUE
+" License: MIT License
+" Last Change: 2015/07/21 13:18:03
+" =============================================================================
+
+fun! <SID>grey_number(x) "{{{
+    if &t_Co == 88
+      if a:x < 23
+        return 0
+      elseif a:x < 69
+        return 1
+      elseif a:x < 103
+        return 2
+      elseif a:x < 127
+        return 3
+      elseif a:x < 150
+        return 4
+      elseif a:x < 173
+        return 5
+      elseif a:x < 196
+        return 6
+      elseif a:x < 219
+        return 7
+      elseif a:x < 243
+        return 8
+      else
+        return 9
+      endif
+    else
+      if a:x < 14
+        return 0
+      else
+        let l:n = (a:x - 8) / 10
+        let l:m = (a:x - 8) % 10
+        if l:m < 5
+          return l:n
+        else
+          return l:n + 1
+        endif
+      endif
+    endif
+  endfun"}}}
+
+  " Returns the actual grey level represented by the grey index
+  fun! <SID>grey_level(n)"{{{
+    if &t_Co == 88
+      if a:n == 0
+        return 0
+      elseif a:n == 1
+        return 46
+      elseif a:n == 2
+        return 92
+      elseif a:n == 3
+        return 115
+      elseif a:n == 4
+        return 139
+      elseif a:n == 5
+        return 162
+      elseif a:n == 6
+        return 185
+      elseif a:n == 7
+        return 208
+      elseif a:n == 8
+        return 231
+      else
+        return 255
+      endif
+    else
+      if a:n == 0
+        return 0
+      else
+        return 8 + (a:n * 10)
+      endif
+    endif
+  endfun"}}}
+
+  " Returns the palette index for the given grey index
+  fun! <SID>grey_colour(n)"{{{
+    if &t_Co == 88
+      if a:n == 0
+        return 16
+      elseif a:n == 9
+        return 79
+      else
+        return 79 + a:n
+      endif
+    else
+      if a:n == 0
+        return 16
+      elseif a:n == 25
+        return 231
+      else
+        return 231 + a:n
+      endif
+    endif
+  endfun"}}}
+
+  " Returns an approximate colour index for the given colour level
```

Comment:
Please replace `colour` with `color` because I consistently use color in this project.